packages/beachball/src/types/ChangeInfo.ts

```diff
@@ -5,9 +5,12 @@ export interface ChangeInfo {
   comment: string;
   packageName: string;
   email: string;
-  commit: string;
   date: Date;
   dependentChangeType?: ChangeType;
 }
 
-export type ChangeSet = Map<string, ChangeInfo>;
+export interface FullChangeInfo extends ChangeInfo {
```

Comment:
how about rename `ChangeInfo` to `ChangeFileInfo` and keep `ChangeInfo` to be the one contains `commit`